### PR TITLE
LESQ-264 Feat/home page tabs

### PR DESCRIPTION
**Pull request by benprotheroe**
## Description

- List of changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-1909--oak-web-application.netlify.thenational.academy/teachers
2. Click on _______
3. You should see _______

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner

### CODE REVIEW

File: src/pages/[viewType]/index.tsx [hunk 1]
```diff
@@ -46,113 +45,21 @@ export type TeachersHomePageProps = HomePageProps & {
...
-            </Grid>
-          </Box>
-        </MaxWidth>
-      </Flex>
+      <HomePageTabImageNav current={current} setCurrent={setCurrent} />
+      {current === "teachers" && (
+        <TeachersTab keyStages={curriculumData.keyStages} />
+      )}
+      {current === "curriculum" && <CurriculumTab />}
+      {current === "pupils" && <PupilTab />}
```
Comment (njblore) [inline src/pages/[viewType]/index.tsx:62]: can we add some tests for this new behaviour?

File: src/components/SearchForm/SearchForm.tsx [hunk 2]
```diff
@@ -83,6 +83,7 @@ type SearchFormProps = {
   placeholderText: string;
   handleSubmit: ({ searchTerm }: { searchTerm: string }) => void;
   analyticsSearchSource: SearchSourceValueType;
+  placeholder?: string;
```
Comment (njblore) [inline src/components/SearchForm/SearchForm.tsx:86]: what's this for?

File: src/components/ResourceSelectorCard/ResourceSelectorCard.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+const Template: ComponentStory<typeof Component> = (args) => (
+  //   <div style={{ background: "lightGrey", padding: "100px" }}>
+  <Component
+    {...args}
+    icon={"worksheet"}
+    title="Worksheets"
+    angle={2}
+    $top={96}
+  />
+  //   </div>
```
Comment (njblore) [inline src/components/ResourceSelectorCard/ResourceSelectorCard.stories.tsx:24]: 👁️ 👁️

File: src/components/HomePageTabs/TeachersTab/TeachersTab.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
+import { FC } from "react";
+
+import Box from "../../Box/Box";
```
Comment (njblore) [inline src/components/HomePageTabs/TeachersTab/TeachersTab.tsx:3]: sneaky relative import

File: src/components/HomePageTabs/TeachersTab/TeachersTab.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  title: "HomePage/TeachersTab",
+  decorators: [AnalyticsDecorator],
+  component: Component,
+  argTypes: {},
+} as ComponentMeta<typeof Component>;
+
+const Template: ComponentStory<typeof Component> = (args) => (
+  //   <div style={{ background: "lightGrey", padding: "100px" }}>
+  <Component {...args} keyStages={keyStageKeypad.keyStages} />
+  //   </div>
```
Comment (njblore) [inline src/components/HomePageTabs/TeachersTab/TeachersTab.stories.tsx:19]: 👁️ 👁️

File: src/common-lib/urls/urls.ts [hunk 6]
```diff
@@ -585,6 +591,12 @@ export const OAK_PAGES: {
     configType: "internal",
     pageType: "curriculum-landing-page",
   }),
+  "early-release-units-page": createOakPageConfig({
+    pathPattern: "/beta/teachers-2023/early-release-units",
```
Comment (njblore) [inline src/common-lib/urls/urls.ts:595]: this has also been done [in my branch](https://github.com/oaknational/Oak-Web-Application/pull/1908/files#diff-deaf037896e78be30b3fdf06dec81c6f72870d073f5f1fabf655731ff7e62c90) but moved out of `/beta` - shall we align them before merging?

**Comment by Monika-Koziol** [comment]
@benprotheroe coming along really well. Here are a few comments:

- [ ] 1. Teaching resources icon: it looks squashed. Can is be in proportions? 
- [ ] 2. White line under the tabs: change thickness to 2px, move down so it looks as if it seats behind the black line of the tabs
- [ ] 3. Small label above the title: ‘Teachers’ etc. should use dark grey colour: oak grey-5
- [ ] 4. “Quizzes” tile: the icon should be fully white, it seems to have a black stroke. Can the tile be positioned a bit lower down and more to the right so there is a bigger gap between ‘worksheets’ and ‘quizzes’
- [ ] 5. Resource tiles all: there is white background sticking outside of the boxes, can that be inside? 
- [ ] 6. ‘Bell icon’ under curriculum plans: can is be fully black and the bell be bigger. It’s ok to update this component to that new style. It also needs a bit of padding on the left as at smaller size is overlapping the early release button
- [ ] 7. Mobile curriculum plans and pupils: the bottom padding is 48px in total. There was a box padding of 24px and the last button had bottom padding of 24px so it was matching the spacing of the first tab with the key pa

d.

**Review by jack-skerrett-oak — CHANGES_REQUESTED** [review]
I will leave the styling stuff between yourself and Monika for the time being, just some a11y changes please!
- [x] Need an aria hidden or something on the bell svg on the curric tab
- [ ] Could we add an aria-label or something to tell a screen reader user that the classroom link will open in a new tab "Learn with Oak (this will open in a new tab)"

File: src/components/SearchForm/SearchForm.tsx (hunk 2 above)
Comment (benprotheroe) [reply src/components/SearchForm/SearchForm.tsx:86]: yep, we added placeholdertext and placeholder at the same time independantly, definitely only need one of them

File: src/__tests__/pages/[viewType]/index.test.tsx [hunk 7]
```diff
@@ -55,9 +55,23 @@ describe("pages/beta/[viewType]/index.tsx", () => {
     render(<Teachers {...props} />);
 
     const h1 = screen.getByRole("heading", { level: 1 });
-    expect(h1).toHaveTextContent("Your foundation for great lessons");
+    expect(h1).toHaveTextContent("Teachers");
   });
+  it.only("Render correct tab after selecting tab", () => {
```
Comment (njblore) [inline src/__tests__/pages/[viewType]/index.test.tsx:60]: is this `.only` intentional`?

**Review by njblore — APPROVED** [review]

**Comment by Monika-Koziol** [comment]
One last little change:

- [ ] Button icon in the pupil tab should be external rather than an arrow as it takes user outside of this experience. (sorry that icon was causing some problems in figma and changed to black)

**Pull request merged**